sdk/eventhub/azure-eventhub/tests/livetest/synctests/test_properties.py

```diff
@@ -17,6 +18,37 @@ def test_get_properties(live_eventhub):
     assert properties['eventhub_name'] == live_eventhub['event_hub'] and properties['partition_ids'] == ['0', '1']
     client.close()
 
+@pytest.mark.liveTest
+def test_get_properties_with_auth_error_sync(live_eventhub):
```

Comment:
I tested in python 37 on a Mac computer. This test method constantly got segment fault. It doesn't happen on other platform. I suggest skipping this test on Mac platform and create a uamqp issue to follow up.
I'll send you the uamqp error msg separately.

Reply:
Turned out to be related to a known logging issue on OSX